ProcessLib/HT/HTProcess.cpp

```diff
@@ -291,66 +290,66 @@ void checkMPLProperties(MeshLib::Mesh const& mesh,
                         HTProcessData const& process_data)
 {
     DBUG("Check the media properties of HT process ...");
+
+    std::vector<MaterialPropertyLib::PropertyType> const reqiredPropertyMedium =
+        {MaterialPropertyLib::PropertyType::porosity,
+         MaterialPropertyLib::PropertyType::permeability};
+
+    std::vector<MaterialPropertyLib::PropertyType> const
+        reqiredPropertyLiquidPhase = {
+            MaterialPropertyLib::PropertyType::viscosity,
+            MaterialPropertyLib::PropertyType::density,
+            MaterialPropertyLib::PropertyType::specific_heat_capacity};
+
+    std::vector<MaterialPropertyLib::PropertyType> const
```

Comment:
An `array` would be better.

Reply:
I thought the same, but then decided for a vector, since the array initialization requires not just the components (density, viscosity,...) but also the array size, which is somehow redundant. Using a vector, one just have to fill in the required properties, letting the compiler count the size.

Reply:
is a const vector that different from an array?

Reply:
vector allocates memory dynamically, but array's size is known at compile time.